pkg/auth/tokens/token_util.go

```diff
@@ -154,6 +165,11 @@ func VerifyToken(storedToken *v3.Token, tokenName, tokenKey string) (int, error)
 	if IsExpired(*storedToken) {
 		return http.StatusGone, errors.New("must authenticate")
 	}
+
+	currentTime := metav1.Now()
+	if IsIdleExpired(*storedToken, currentTime) {
+		return http.StatusGone, errors.New("must authenticate, idle session timeout expired")
+	}
```

Comment:
Not sure why we can't inline it. Not that this additional allocation makes it any easier to read/understand.
```suggestion
	if IsIdleExpired(*storedToken, metav1.Now()) {
		return http.StatusGone, errors.New("must authenticate, idle session timeout expired")
	}
```